stream/tag_mapper.js

```diff
@@ -50,7 +50,16 @@ module.exports = function(){
             if( key === NAME_SCHEMA._primary ){
               doc.setName( NAME_SCHEMA[key], val2 );
             } else if ( 'default' === NAME_SCHEMA[key] ) {
-              doc.setNameAlias( NAME_SCHEMA[key], val2 );
+              // `alt_name` may contain multiple names separated by semicolons 
+              // (see https://wiki.openstreetmap.org/wiki/Key:alt_name#Examples)
+              if (key === 'alt_name') {
```

Comment:
Tbh at glance it seems to be better to "bake" the fact that we need to split by something to `NAME_SCHEMA`, but it seems to be a bit of over-engineering for now (but let me know :) )